main/urls.py

```diff
@@ -1,6 +1,17 @@
-from django.urls import path
+from django.urls import path, include
 
 from main.views import home
+from main.api import ConfigurationView
+from rest_framework import routers
+
+api_router = routers.DefaultRouter()
+# api/main/configuration
+api_router.register('configuration', ConfigurationView)
```

Comment:
Please use plural names in resources URLs

```suggestion
api_router.register('configurations', ConfigurationView)
```